uninstall/3.1/uninstall-cp4waiops-helper.sh

```diff
@@ -410,11 +412,11 @@ log $INFO "##### Properties in uninstall-cp4waiops-props.sh #####"
 check_additional_installation_exists(){
 
   log $INFO "Checking if any additional installation resources found in the cluster."
-  installation_returned_value=$(oc get installations.orchestrator.aiops.ibm.com -A)
+  installation_returned_value=$(oc get installation -A)
```

Comment:
I think we want this to still be `installations.orchestrator.aiops.ibm.com` instead of `installation`